srml/staking/src/phragmen.rs

```diff
@@ -269,21 +269,21 @@ pub fn elect<T: Trait + 'static, FV, FN, FS>(
 	Some((elected_candidates, assigned))
 }
 
-/// Performs equalize post-processing to the output of the election algorithm
+/// Performs equalise post-processing to the output of the election algorithm
 /// This function mutates the input parameters, most noticeably it updates the exposure of
 /// the elected candidates.
 ///
 /// No value is returned from the function and the `expo_map` parameter is updated.
-pub fn equalize<T: Trait + 'static>(
-	assignments: &mut Vec<(T::AccountId, BalanceOf<T>, Vec<Assignment<T>>)>,
+pub fn equalise<T: Trait + 'static>(
+	assignments: &mut Vec<(T::AccountId, BalanceOf<T>, Vec<(T::AccountId, ExtendedBalance, ExtendedBalance)>)>,
```

Comment:
I'm not a big fan of such complicated tuples, maybe a `type definition` could make sense?

Reply:
We already have `type Assignment<T>` and `type RawAssignment<T>`. I think more than this would be tipping from the other side: _too many_ types. Sadly there are some places that we want to collect, for example, a Vec of assignments with a modified field, so the type will no longer work. I would settle for what is already there for now. Maybe more refactoring would make it possible in the future.

Reply:
Ok, I see